tests/tools/kopiarunner/kopiarun.go

```diff
@@ -72,53 +70,12 @@ func (kr *Runner) Run(args ...string) (stdout, stderr string, err error) {
 	c := exec.Command(kr.Exe, cmdArgs...)
 	c.Env = append(os.Environ(), kr.environment...)
 
-	stderrPipe, err := c.StderrPipe()
-	if err != nil {
-		return stdout, stderr, err
-	}
-
-	var wg sync.WaitGroup
-
-	wg.Add(1)
-
-	errOut := bytes.Buffer{}
-
-	go func() {
-		defer wg.Done()
-
-		scanner := bufio.NewScanner(stderrPipe)
-		for scanner.Scan() {
-			log.Println(scanner.Text())
-			errOut.Write(scanner.Bytes())
-			errOut.WriteByte('\n')
-		}
-	}()
-
-	stdoutPipe, err := c.StdoutPipe()
-	if err != nil {
-		return stdout, stderr, err
-	}
-
-	wg.Add(1)
-
-	o := bytes.Buffer{}
-
-	go func() {
-		defer wg.Done()
-
-		scanner := bufio.NewScanner(stdoutPipe)
-		for scanner.Scan() {
-			log.Println(scanner.Text())
-			o.Write(scanner.Bytes())
-			o.WriteByte('\n')
-		}
-	}()
-
-	err = c.Run()
+	errOut := &bytes.Buffer{}
+	c.Stderr = errOut
 
-	wg.Wait()
+	o, err := c.Output()
 
-	log.Printf("finished '%s %v' with err=%v and output:\nSTDOUT:\n%v\nSTDERR:\n%v", kr.Exe, argsStr, err, o.String(), errOut.String())
+	log.Printf("finished '%s %v' with err=%v and output:\nSTDOUT:\n%v\nSTDERR:\n%v", kr.Exe, argsStr, err, string(o), errOut.String())
```

Comment:
Any reason to log the output and also return it?

Reply:
The log of every command status has proven to be helpful for debug (similar to the kopia e2e tests, which follows the same pattern for the command runner), and the outputs are returned for parsing (for instance getting the latest snapshot from manifest list)